imap_processing/codice/constants.py

```diff
@@ -78,112 +78,271 @@
 # TODO: Hi products shape should be energy x ssd index x spin sector (8*12*12)
 DATA_PRODUCT_CONFIGURATIONS = {
     CODICEAPID.COD_HI_INST_COUNTS_AGGREGATED: {
+        "coords": [
+            "epoch",
+            "inst_az",
+            "spin_sector",
+            "esa_step",
+            "energy_label",
+        ],  # TODO: These will likely change
+        "dataset_name": "imap_codice_l1a_hi_counters_aggregated",
```

Comment:
This seems to follow filename convention. If so, can you change this and others to use `-` instead of `_` for the descriptor names?
```suggestion
        "dataset_name": "imap_codice_l1a_hi_counters-aggregated",
```

Reply:
Good idea to be consistent. I think I went down the road of using underscored because of how I defined things in the CDF attr config YAML files. Since the `dataset_name` is being used to find the appropriate key in those files, I will also need to update from underscore to hyphen there too.

Just pointing this out so that it is clear why those changes will be in the next commit to this PR.